pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -156,6 +159,10 @@ stdenv.mkDerivation rec {
     "-DARROW_COMPUTE=ON"
     "-DARROW_CSV=ON"
     "-DARROW_DATASET=ON"
+    "-DARROW_ENGINE=ON"
+    "-DARROW_FILESYSTEM=ON"
+    "-DARROW_FLIGHT_SQL=ON"
```

Comment:
```suggestion
    "-DARROW_FLIGHT_SQL=${if enableFlight then "ON" else "OFF"}"
```

Reply:
Whoops, thanks 😅

Reply:
Still needs fixing!

Reply:
Hm, I thought I committed this. I must've forced pushed from an older commit. Fixing now.